base/changes.txt

```diff
@@ -6,6 +6,10 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2025-01-26  Joseph Wright  <Joseph.Wright@latex-project.org>
+	* ltexpl.dtx, usrguide.tex
+	Add \expandableinput
```

Comment:
There was the suggestion to use \ExpandableInput, not sure which is better or if there should/could be adifferent name

Reply:
I wasn't sure if the name suggestion was all-lower or CamelCase - as this feels like a document command ...

Reply:
Are "document commands" the CamelCase ones?